lang/cs/Org.Apache.REEF.Client/YARN/RESTClient/DataModel/ServiceData.cs

```diff
@@ -22,10 +24,8 @@ namespace Org.Apache.REEF.Client.YARN.RestClient.DataModel
     /// <see cref="!:http://hadoop.apache.org/docs/r2.6.0/hadoop-yarn/hadoop-yarn-site/WebServicesIntro.html">
     /// Hadoop RM REST API </see> documentation.
     /// </summary>
-    internal sealed class MaximumResourceCapability
+    internal sealed class ServiceData
     {
-        public int Memory { get; set; }
-
-        public int VCores { get; set; }
+        public KeyValuePair<string, string> Entry { get; set; }
```

Comment:
I am not sure about that.
From the documentation:
service-data: Application specific service data; key is the name of the auxiliary servce, value is base-64 encoding of the data you wish to pass

Where as when something was a list the documentation said:
Elements of the local-resources object: The object is a collection of key-value pairs.